src/Core/Enums/DisableSendPolicyType.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace Bit.Core.Enums
+{
+    public enum DisableSendPolicyType : byte
+    {
+        DisableAll = 0,
+        DisableAnonymous = 1,
```

Comment:
I think this policy item is valid, but I also think it can be misconstrued as applying to access.

We currently can't regulate Send Access at all, but I think we need to be careful how this is explained

Reply:
One more thought on this. The Send isn't truly anonymous in the fact that we could trace back who created it. I think for our product, anonymous should refer to true zero knowledge, not just that we don't share it. This is more of a documentation note, I think, but a variable name tweak might be in order too

Reply:
This type will likely need to go away altogether.

Reply:
Good point - I'll use HideEmail as the verbiage rather than anonymous (which feels bad to type on qwerty anyway)

Reply:
`DisableHideCreatorIdentifier` or something to that effect in the property on the policy data itself, yes, this enum would go away all-together however as it is no longer really needed.